app/templates/components/ui-table/cell/admin/events/event-is-featured.hbs

```diff
@@ -1 +1,6 @@
-<UiCheckbox @class="toggle" @checked={{this.extraRecords.isFeatured}} @onChange={{action this.props.actions.toggleFeatured this.record}} />
+<XToggle
+  @theme='ios'
+  @variant='dark'
+  @value={{this.extraRecords.isFeatured}}
+  @onToggle={{action this.props.actions.toggleFeatured this.record}}
+/>
```

Comment:
We need to modify this in one place rather than one by one

I am working on the issue here, you can continue on that PR

Reply:
@iamareebjamal , can you share the link to the PR?

Reply:
https://github.com/fossasia/open-event-frontend/pull/5268

Reply:
Perfect, I will push some commits there.

Reply:
@iamareebjamal , I have made some changes, let me know if we require something more.